src/Migrations/Cli/Command/ExecuteMigrationCommand.php

```diff
@@ -125,8 +125,6 @@ protected function execute(InputInterface $input, OutputInterface $output)
             $direction
         );
 
-        if (!$result) {
-            return 1;
-        }
+        return $result ? 0 : 1;
```

Comment:
Can use the constants here too Command::SUCCESS and Command::FAILURE

Reply:
There are a whole lot of places with similar stuff will go over everything I can find and try to refactor it